lib/app.tsx

```diff
@@ -234,7 +234,6 @@ export const App = connect(
         this.props.openTagList();
 
         event.stopPropagation();
-        event.preventDefault();
```

Comment:
if we do this, won't we _always_ re-open the last-closed tab if we can? I don't think we want that either.

it seems like it could be a good idea to consider a different keybinding. I spoke with @belcherj about this yesterday and we didn't have any good ideas, what do you think about changing it?

it's the "tag list" or the "group list" or the "side sidebar"

<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>o</kbd>

`o` for "organization" ?
<del>`c` for "categorization" ?</del> (dev-tools select element)
`e` because it's available?
`m` because it's available?

Reply:
Yeah, it's kind of annoying. On the Mac you can choose to use Ctrl in the browser and not Cmd (and I wish we could just exclude Cmd from being valid). On Windows I guess you're stuck. I would be open to changing it.